mlflow/store/model_registry/file_store.py

```diff
@@ -905,6 +955,18 @@ def _read_helper(root, file_name, attempts_remaining=2):
 
         return _read_helper(root, file_name, attempts_remaining=retries)
 
+    def copy_model_version(self, mv, dst_name) -> ModelVersion:
```

Comment:
What happens if we copy against teh same registered model?

Reply:
Yes. This will just create a new model version in the same registered model. I don't see any customer use-cases for doing so but I also don't see any necessity in implementing logic to prevent it.